<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>9</y>
      <width>561</width>
      <height>401</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Course: </string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QComboBox" name="dropdown_box_2">
           <item>
            <property name="text">
             <string>PIC 10B: Intermediate Programming</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>PIC 10C: Advanced Programming</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="dropdown_box">
           <item>
            <property name="text">
             <string>Schema A</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Schema B</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLabel" name="hw1_label">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Hw 1</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="hw1_slider">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="hw1_score">
             <property name="maximum">
              <double>100.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="hw2_label">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Hw 2</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="hw2_slider">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="hw2_score">
             <property name="maximum">
              <double>100.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="hw3_label">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Hw 3</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="hw3_slider">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="hw3_score">
             <property name="maximum">
              <double>100.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="hw4_label">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Hw 4</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="hw4_slider">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="hw4_score">
             <property name="maximum">
              <double>100.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="hw5_label">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Hw 5</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="hw5_slider">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="hw5_score">
             <property name="maximum">
              <double>100.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="hw6_label">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Hw 6</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="hw6_slider">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="hw6_score">
             <property name="maximum">
              <double>100.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="hw7_label">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Hw 7</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="hw7_slider">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="hw7_score">
             <property name="maximum">
              <double>100.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="hw8_label">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Hw 8</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="hw8_slider">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="hw8_score">
             <property name="maximum">
              <double>100.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="mid1_label">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Midterm 1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="mid1slider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="mid1_score">
           <property name="maximum">
            <double>100.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="mid2_label">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Midterm 2</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="mid2slider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="mid2_score">
           <property name="maximum">
            <double>100.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="final_label">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Final Exam</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="finalslider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="final_score">
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="maximum">
            <double>100.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="OveralScore_label">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Overal Score:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="score_output_label">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>0.0</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>hw1_label</zorder>
   <zorder>hw2_label</zorder>
   <zorder>hw3_label</zorder>
   <zorder>hw4_label</zorder>
   <zorder>hw5_label</zorder>
   <zorder>hw6_label</zorder>
   <zorder>hw7_label</zorder>
   <zorder>hw8_label</zorder>
   <zorder>hw1_slider</zorder>
   <zorder>hw2_slider</zorder>
   <zorder>hw3_slider</zorder>
   <zorder>hw4_slider</zorder>
   <zorder>hw5_slider</zorder>
   <zorder>hw6_slider</zorder>
   <zorder>hw7_slider</zorder>
   <zorder>hw8_slider</zorder>
   <zorder>hw1_score</zorder>
   <zorder>hw2_score</zorder>
   <zorder>hw3_score</zorder>
   <zorder>hw4_score</zorder>
   <zorder>hw5_score</zorder>
   <zorder>hw6_score</zorder>
   <zorder>hw7_score</zorder>
   <zorder>hw8_score</zorder>
   <zorder>mid1_label</zorder>
   <zorder>mid2_label</zorder>
   <zorder>final_label</zorder>
   <zorder>mid1slider</zorder>
   <zorder>mid2slider</zorder>
   <zorder>finalslider</zorder>
   <zorder>mid1_score</zorder>
   <zorder>mid2_score</zorder>
   <zorder>final_score</zorder>
   <zorder>OveralScore_label</zorder>
   <zorder>score_output_label</zorder>
   <zorder>label</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGrade_Calculator">
    <property name="title">
     <string>Grade Calculator</string>
    </property>
   </widget>
   <addaction name="menuGrade_Calculator"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
